nixos/modules/services/desktops/pipewire/pipewire.nix

```diff
@@ -251,6 +251,8 @@ in {
         ] ++ lib.optional config.security.rtkit.enable "rtkit";
         description = "Pipewire system service user";
         isSystemUser = true;
+        home = "/var/lib/pipewire";
+        createHome = true;
```

Comment:
We should set homeMode.

Isn't it enough to create this directory with systemds StateDirectory.

Reply:
We're using upstream units and they don't set it, unfortunately. I guess we could patch the units? Also, homeMode 0700 should be fine, though possibly unnecessary.

Reply:
Does NixOS merge the systemd files if we set the option in systemd.services ?

Reply:
I don't think so, though systemd _might_ by accident.

Reply:
Okay, fine for me. We should still set homeMode to something other than 755.

Reply:
The default is 700, not 755.

Reply:
(but also oh god someone stole my speaker volume whatever should I do)

Reply:
Oh, you're right! Thanks for clarifying.